sdk/spring/azure-spring-boot-core/pom.xml

```diff
@@ -49,6 +49,11 @@
       <version>2.5.0</version> <!-- {x-version-update;org.springframework.boot:spring-boot-starter-test;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
Why do we need this?

Reply:
This is used to import @PostConstruct, I will change it to jakarta.annotation:jakarta.annotation-api

Reply:
no, as the core lib, we should not have this core lib depends on spring-boot-starter-web directly